docs/tutorials/digital-twin-simulation/awsim-tutorial.md

```diff
@@ -0,0 +1,77 @@
+# AWSIM Simulator Setup
+
+This instruction presents how to run Tier IVs AWSIM with Autoware.
+
+## Preparation
+
+- Install Autoware following the official instruciton : [link](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/)
+- Download and extract AWSIM binary : [link](https://drive.google.com/file/d/1jgB_A5cSso9dzcNffQYthe07GG07QLQh/view?usp=sharing)
+- Download and extract map folder : [link](https://drive.google.com/file/d/1vGFI0o0zQ-gRZYqKrPbnrtCN3c3-92Fy/view?usp=sharing)
+
+## Running steps
+
+### AWSIM Simulator
+
+The AWSIM Simulator can be run without any further preparation. To run the simulator:
+
+- open the terminal navigate to the simulation directory and run:
+
+        source /opt/ros/galactic/setup.bash
+        cd <PATH_TO_EXTRACTED_FOLDER_WITH_AWSIM>
+        ./AWSIM.x86_64
+
+The properly working simulator should have the lidar rays visible and the data topics should appear when discovering using `ros2 topic list`.
+
+![correctly working simulator](./images/workingSim.png)
+
+#### Simulator Shourtcuts
+
+It is possible to adjust/control the simulation through the shortcuts listed below:
+
+- v - turn on/off lidar points visualization
+- m - enter/exit manual operation mode (Ego listens to keyboard)
+  - In manual mode:
+    - d - drive
+    - p - parking brake
+    - r - reverse
+    - arrows - acceleration and steering
+    - lights:
+      - 1 - left blinker
+      - 2- right blinker
+      - 3 - emergency lights
+      - 4- turning the lights off
+
+### Autoware
```

Comment:
The title of this section does not describe its purpose. Is it correct to say that this section describes how to use Autoware to control AWSIM?